app/controllers/list_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ListController < ApplicationController
+  def index
+    @lists = List.all
+  end
+
+  def edit
+    @list = List.find_by_id(params[:id])
+  end
+
+  def new
+    @list = List.new
+  end
+
+  def show
+    @list = List.find_by_id(params[:id])
+  end
+
+  def create
+    @list = List.new(list_params.merge(therapist: current_user.role))
+    if @list.save
+      render json: {list: @list}, status: :ok
+    else
+      render json: {errors: @list.errors.full_messages}, status: :bad_request
+    end
+  end
+
+  def update
+    @list = List.find_by_id(params[:id])
```

Comment:
See above re: JSON API, same best practices, but re-display the `edit` page on failure.